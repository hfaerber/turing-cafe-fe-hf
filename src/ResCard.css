.cards {
  background-color: white;
  border: 2px solid teal;
}
