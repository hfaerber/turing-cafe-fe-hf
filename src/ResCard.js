import React from 'react'
import './ResCard.css'

const ResCard = ({ name, id, date, time, number, deleteRes }) => {
  return(
    <section className='cards'>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time}</p>
      <p>Number of guests: {number}</p>
      <button onClick={() => deleteRes(id)}>Cancel</button>
    </section>
  )
}

export default ResCard
