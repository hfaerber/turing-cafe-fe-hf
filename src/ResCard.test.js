import React from 'react'
import { shallow } from 'enzyme';
import ResCard from './ResCard';

  describe('ResCard', () => {

    it('should match snapshot with all data passed through correctly', () => {
      const wrapper = shallow(<ResCard
          key={11}
          id={11}
          name='Robbie'
          date='04/11'
          time='7:45 pm'
          number={2}
          deleteRes={jest.fn()}
        />)
      expect(wrapper).toMatchSnapshot();
    })


})
