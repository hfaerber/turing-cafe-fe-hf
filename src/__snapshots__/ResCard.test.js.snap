// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResCard should match snapshot with all data passed through correctly 1`] = `
<section
  className="cards"
>
  <h2>
    Robbie
  </h2>
  <p>
    04/11
  </p>
  <p>
    7:45 pm
  </p>
  <p>
    Number of guests: 
    2
  </p>
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</section>
`;
